gravitee-apim-console-webui/src/management/api/api-traffic-v4/analytics/api-analytics-proxy/api-analytics-proxy.component.ts

```diff
@@ -22,23 +22,35 @@ import { CommonModule } from '@angular/common';
 
 import { ApiAnalyticsFiltersBarComponent } from '../components/api-analytics-filters-bar/api-analytics-filters-bar.component';
 import { AggregationFields, AggregationTypes } from '../../../../../entities/management-api-v2/analytics/analyticsHistogram';
-import { ChartWidgetConfig } from '../../../../../entities/management-api-v2/analytics/analytics';
-import { ChartWidgetComponent } from '../components/chart-widget/chart-widget.component';
+import { WidgetConfig } from '../../../../../entities/management-api-v2/analytics/analytics';
+import { WidgetComponent } from '../components/widget/widget.component';
 
 @Component({
   selector: 'api-analytics-proxy',
-  imports: [CommonModule, MatCardModule, GioLoaderModule, GioCardEmptyStateModule, ApiAnalyticsFiltersBarComponent, ChartWidgetComponent],
+  imports: [CommonModule, MatCardModule, GioLoaderModule, GioCardEmptyStateModule, ApiAnalyticsFiltersBarComponent, WidgetComponent],
   templateUrl: './api-analytics-proxy.component.html',
   styleUrl: './api-analytics-proxy.component.scss',
 })
 export class ApiAnalyticsProxyComponent {
-  public chartWidgetConfigs: ChartWidgetConfig[] = [
+  public tableWidgets: WidgetConfig[] = [
+    {
+      type: 'table',
+      apiId: this.activatedRoute.snapshot.params.apiId,
+      title: 'Top Applications',
+      tooltip: 'Apps ranked by total API calls over time',
```

Comment:
I would say Applications and not Apps here. I'm not a big fan of acronyms in tooltips
```suggestion
      tooltip: 'Applications ranked by total API calls over time',
```